keps/sig-node/2400-node-swap/README.md

```diff
@@ -117,6 +124,20 @@ support to nodes in a controlled, predictable manner so that Kubernetes users
 can perform testing and provide data to continue building cluster capabilities
 on top of swap.
 
+This KEP aims to
+introduce basic swap enablement and leave further extensions to follow-up KEPs.
+This way Kubernetes users / vendors would be able to use swap in a basic manner
+quickly while extensions would be brought to discussion in dedicated KEPs that
+would progress in the meantime.
+
+For example, to achieve this goal, this KEP does not introduce any APIs
+that allow customizing how the feature behaves, but instead only determines 
+whether the feature is enabled or disabled.
+Instead, the behaviour is automatic and implicit that requires minimum user
+intervention (see [proposal below](#steps-to-calculate-swap-limit) for more details).
+As mentioned above, in the very near future, follow-up KEPs would bring API extension
```

Comment:
Per the feedback from the SIG-Node meeting today, it's good to have early exploration to make sure whatever we GA is compatible with the future changes. If you already have some thoughts on this, maybe you could add a few sentences here. @iholder101

Reply:
I've reshaped this section, mentioning that currently we have only two on/off swap behaviors with no APIs or customizability (`NoSwap` and `LimitedSwap`), and that in the future we'll add customizability through adding more swap behaviors that might lead to API changes, perhaps at the pod level.

It was agreed many times before that this KEP would revolve around basic swap enablement. I've intentionally left this vague as API changes is a controversial, complex and serious topic, and I prefer to have proper discussions about it in a follow-up KEP. As agreed upon in yesterday's sig-node meeting, I'll start working on a sketch document to initiate these conversations which will serve as a first step for a follow-up KEP.

PTAL